recipes/systemd/meta.yaml

```diff
@@ -0,0 +1,136 @@
+{% set name = "systemd" %}
+{% set version = "243" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/systemd/systemd/archive/v{{ version }}.tar.gz
+  sha256: 0611843c2407f8b125b1b7cb93533bdebd4ccf91c99dffa64ec61556a258c7d1
+  patches:
+    # fixes compilation issue for missing socket decl
+    # from https://bugs.freedesktop.org/show_bug.cgi?id=67075
+    - l2tp.h.patch
```

Comment:
Could these be moved to a `patches` folder to make it easier to find the actual recipe?